aws-ssm-patchbaseline/aws-ssm-patchbaseline.json

```diff
@@ -280,14 +280,19 @@
         "Name"
     ],
     "createOnlyProperties": [
-        "/properties/OperatingSystem"
+        "/properties/OperatingSystem",
```

Comment:
+1, why are defaultBaseline/Tags CreateOnly, customer should be able to alter those values, and that's our current behavior

Reply:
1) "/properties/OperatingSystem", - It will help add the Operation details for the patchbaseline and returns the same
 2) "/properties/DefaultBaseline", - It is an optional. It will help to add the defaultbaseline configurations for patch baseline and returns the same.
3)  "/properties/Tags" - It is an optional, It will help to add the tags for the patchbaseline and it returns the tags list

Reply:
yeah, i understand what it means, but i don't understand why do you define them in createOnlyProperties, 
i think createOnlyProperties means customers can never change it once it's created, 
but those 2 properties are updatable.

Reply:
Could you please suggest me those two properties, Can i remove from the createOnlyProperties section and also please suggest me if any other way is available to declare those properties